src/hacbs/components/Modals/resource-modals.tsx

```diff
@@ -0,0 +1,16 @@
+import * as React from 'react';
+import { createDeleteModalLauncher } from '../../../components/modal/DeleteResourceModal';
+import { PipelineRunModel } from '../../models/pipelineruns';
+import { PipelineRunKind } from '../../types';
+
+export const pipelineRunDeleteModal = (pipelineRun: PipelineRunKind) =>
+  createDeleteModalLauncher(pipelineRun.kind)({
+    obj: pipelineRun,
+    model: PipelineRunModel,
+    description: (
+      <>
+        The pipelinerun <strong>{pipelineRun.metadata.name}</strong> will be deleted from the
+        pipelinerun view and all the environments it is currently deployed.
```

Comment:
@mfrances17 I don't think we want to delete pipeline runs. Nor do I believe that deletion of a pipeline run would have such cascading side effects as to remove a deployed snapshot from an environment.

Reply:
The attachment you referred to was my screen grab from OpenShift Console where I was inquiring about the cancel action.

That's fine to log a new story for rerun and do it later.

Reply:
i've removed Delete and will create a new story for Rerun.